compiler/bootstrap/translation/reg_allocProgScript.sml

```diff
@@ -365,18 +400,34 @@ TODO: update the following code (comes from the non-monadic register allocator
 
 misc code to generate the unverified register allocator in SML
 
-open ml_progLib astPP
+(* Not sure where this gets overwritten *)
+val implode = String.implode;
+val explode = String.explode;
+
+open ml_progLib cfLib basis
+open astPP
+
+val main = process_topdecs`
+  fun main u = ()`
+
+val res = append_prog main;
+
+val st =  get_ml_prog_state ();
+
+Theorem main_whole_prog_spec
+  `whole_prog_spec ^(fetch_v "main" st) cl fs NONE (\x. T)`
+  cheat
```

Comment:
This will never be proved, right? Might be better with `mk_thm` or `mk_oracle_thm`.

Reply:
I have tweaked this to assume `F` and then prove it using `simp []`.